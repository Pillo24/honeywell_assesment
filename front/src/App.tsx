import './App.css'

interface AppProps {}

const App: React.FC<AppProps> = () => {
    return (
        <div className="App">
            <header className="App-header">
                <h3>Hello there #FutureShaper...</h3>
                <p>
                    This is a simple React app that will be used to asses your
                    skills.
                </p>
                <p>
                    Please follow the instructions in the README.md file to get
                    started.
                </p>
                <p>
                    Edit <code>src/App.js</code> and show me what you got!
                </p>
            </header>
        </div>
    )
}

export default App
