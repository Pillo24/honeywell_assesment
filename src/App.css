.App {
  align-items: center;
  background-color: #fff;
  display: flex;
  flex: 0 0 100%;
  justify-content: center;
  min-height: 100vh;
}

p {
  font-size: .8em;
  margin: 0;
}

img {
  max-width: 100%;
}