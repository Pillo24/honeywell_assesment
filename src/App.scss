@import url("https://fonts.googleapis.com/css2?family=Open+Sans:wght@300;400;600;800&display=swap");
.App {
  text-align: center;
  min-height: 100vh;
  background-color: #ececec;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-family: "Open Sans", sans-serif;
  .title {
    font-size: calc(10px + 2vmin);
    margin-bottom: 20px;
    color: #505050;
    z-index: 3;
  }

  .box {
    display: flex;
    border-radius: 8px;
    position: relative;
    box-shadow: 20px 20px 60px #bebebe, -20px -20px 60px #ffffff;
    .form_container {
      border-radius: 8px 0 0 8px;
      background-color: #f5f5f5;
      padding: 20px;
      .form_container_title {
        font-size: calc(20px);
        margin-bottom: 20px;
      }
      form {
        font-size: calc(10px + 0.5vmin);
        button {
          background-color: #f5b700;
          border: none;
          border-radius: 4px;
          width: 100px;
          height: 30px;
          cursor: pointer;
          color: #fff;
        }
      }
    }
    .right_site_container {
      background-color: #f5b700;
      border-radius: 0 8px 8px 0;
      width: 300px;
      display: flex;
      flex-direction: column;
      justify-content: center;
      align-items: center;
      .quote_container {
        display: flex;
        align-items: center;
        justify-content: center;
        color: #fff;
        font-weight: 600;
        padding: 20px;
      }
    }
  }
}

@media only screen and (max-width: 600px) {
  .box {
    display: flex;
    flex-direction: column;
    .form_container {
      border-radius: 8px 8px 0 0 !important;
    }
    .right_site_container {
      width: 360px !important;
      border-radius: 0 0 8px 8px !important;
    }
  }
}
