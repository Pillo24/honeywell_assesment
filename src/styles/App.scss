.App {
  text-align: center;
}

.App-header {
  background-color: black;
  min-height: 100vh;
  display: flex;
  flex-direction: column;
  align-items: center;
  justify-content: center;
  font-size: calc(10px + 2vmin);
  font-family: Consolas,monaco,monospace; 
  color: #008F11;
}
.App-body {
    margin: 0;
    height: 100vh;
    width: 100vw;

}
