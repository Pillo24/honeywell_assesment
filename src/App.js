import Panel from "./components/login/Panel";
import Home from "./components/Home/Home";
import "./App.css";
import { BrowserRouter, Routes, Route } from "react-router-dom";

function App() {
  return (
    <BrowserRouter>
      <div className="App">
        <Routes>
          <Route path="/" element={<Panel />} />
          <Route path="/login" element={<Panel />} />
          <Route path="/register" element={<Panel />} />
          <Route path="/home" element={<Home />} />
        </Routes>
      </div>
    </BrowserRouter>
  );
}

export default App;
