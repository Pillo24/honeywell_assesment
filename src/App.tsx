import React, { Suspense } from 'react';
import { BrowserRouter as Router, Routes, Route } from 'react-router-dom';

// Pages
import { SingUpPage } from './pages';  

// Components
import Layout from './components/layout';

// Styles
import './styles/App.scss';

function App() {
  return (
    <Layout>
      <Router>
        <Suspense fallback={<div>Loading...</div>}>
          <Routes>
            <Route path="/" element={<SingUpPage />} />
            <Route path="/login" element={<SingUpPage />} />
          </Routes>
        </Suspense>
      </Router>
    </Layout>
  );
}

export default App;
