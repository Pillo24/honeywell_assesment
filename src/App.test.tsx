import React from 'react';
import { render, screen } from '@testing-library/react';
import App from './App';

describe('App', () => {
  test('renders component', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});