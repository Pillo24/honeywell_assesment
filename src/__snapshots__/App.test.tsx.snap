// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders component 1`] = `
<DocumentFragment>
  <header
    class="header__container"
  />
  <div />
</DocumentFragment>
`;
